Microsoft.Toolkit.Uwp.SampleApp/Shell.xaml

```diff
@@ -6,6 +6,11 @@
       xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
       xmlns:local="using:Microsoft.Toolkit.Uwp.SampleApp"
       xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      xmlns:viewHelper="using:Microsoft.Toolkit.Uwp.UI.Extensions"
```

Comment:
can we change the namespace declaration to viewExtensions or just extensions. "viewHelper" doesn't make sense for the namespace